Documentation for building the command-line client is inadequate

The current instructions at https://github.com/docker/notary/blob/master/README.md say:

> Prerequisites:
> 
> Go >= 1.6.1
> godep installed
> libtool development headers installed
> Ubuntu: apt-get install libtool-dev
> CentOS/RedHat: yum install libtool-ltdl-devel
> Mac OS (Homebrew): brew install libtool
> Run make binaries, which creates the Notary Client CLI binary at bin/notary.

I've tried to follow all these instructions, yet "make binaries" fails with dependency errors:

> GOPATH=/root/go PATH=$PATH:/usr/local/go/bin go get github.com/tools/godep
> GOPATH=/root/go:/root/notary/Godeps/_workspace PATH=$PATH:/usr/local/go/bin make binaries
> pkcs11 import was not found anywhere without a build tag, yay
> xargs: go: terminated by signal 13
> + /root/notary/bin/notary-server
> cmd/notary-server/config.go:9:2: cannot find package "github.com/Sirupsen/logrus" in any of:
> /usr/local/go/src/github.com/Sirupsen/logrus (from $GOROOT)
> /root/go/src/github.com/Sirupsen/logrus (from $GOPATH)
> /root/go/src/github.com/Sirupsen/logrus
> /root/notary/Godeps/_workspace/src/github.com/Sirupsen/logrus

Could someone update the docs with the full list of requirements?  Thanks.

@riyazdf 

> > go version
> go version go1.6.1 linux/amd64
> 

> > git log -n 1
> commit 358add60750328a5dea72c5cf43e29eacd9e6e97
> Merge: ebb2eb8 84b1210
> Author: David Lawrence <dclwrnc@gmail.com>
> Date:   Tue Apr 19 15:45:45 2016 -0700
> 
>     Merge pull request #686 from docker/go1.6.1
> 
>     Update go in dockerfiles to go 1.6.1
> 
> > GOPATH=$(pwd) make binaries
> xargs: go: terminated by signal 13
> pkcs11 import was not found anywhere without a build tag, yay
> + /root/notary/bin/notary-server
> cmd/notary-server/config.go:9:2: cannot find package "github.com/Sirupsen/logrus" in any of:
>         /usr/local/go/src/github.com/Sirupsen/logrus (from $GOROOT)
>         /root/notary/src/github.com/Sirupsen/logrus (from $GOPATH)

--- a/README.md
+++ b/README.md
@@ -91,8 +91,7 @@ Run `make binaries`, which creates the Notary Client CLI binary at `bin/notary`.
 Note that `make binaries` assumes a standard Go directory structure, in which
 Notary is checked out to the `src` directory in your `GOPATH`. For example:
 ```
-GOPATH=$(<path to go/>)
-go/
+$GOPATH/
     src/
         github.com/
             docker/

--- a/README.md
+++ b/README.md
@@ -88,3 +88,13 @@ Prerequisites:
     - Mac OS ([Homebrew](http://brew.sh/)): `brew install libtool`
 
 Run `make binaries`, which creates the Notary Client CLI binary at `bin/notary`.
+Note that `make binaries` assumes a standard Go directory structure, in which
+Notary is checked out to the `src` directory in your `GOPATH`. For example:
+```
+GOPATH=$(<path to go/>)
+go/
+    src/
+        github.com/
+            docker/
+                notary/
+```

